varia/sass/child-theme/_global-variables.scss

```diff
@@ -0,0 +1,25 @@
+@mixin global-variables() {
+
+	/* Colors */
+	--wp--preset--color--primary: blue;
```

Comment:
It seems to me that these values, rather than attempting to copy the function of a "palette" and named such (--wp--preset--color) should instead be CUSTOM values and named according to that...

--wp--custom--color--primary   (etc)

As a way to reflect values that might be changed (variables) as opposed to values that can't (palettes).

Reply:
I see what you mean, but for these themes, will we ever have a preset palette? what values would it have?

Reply:
I'm not sure I agree with past-pbking.